common/locales/en-US/main.ftl

```diff
@@ -250,7 +250,9 @@ settings-about = About Settings
     .open-website-description = Opens our website in your default web browser.
     .open-codebase = Open Source Code
     .open-codebase-description = Opens the codebase in your default web browser.
-
+    .no-update-available = Your software is up to date!
+    .update-check-error = There was an error checking for the update. Please report this!
```

Comment:
this will also appear if user doesn't have internet, so maybe in the future we could add a error just for the internet connection